app/src/main/java/com/kms/cura/view/activity/PatientViewActivity.java

```diff
@@ -48,6 +48,11 @@ private void initFragments() {
         patientProfileFragment = new PatientProfileFragment();
         patientHealthTrachkerFragment = new HealthTrackerFragment();
         patientAppointmentFragment = new PatientAppointmentListFragment();
+        if (getIntent().getStringExtra(NAVIGATION_KEY) != null && getIntent().getStringExtra(NAVIGATION_KEY).equals(ConditionSymptomSearchActivity.TO_HEALTH_TRACKER)) {
```

Comment:
extract getIntent().getStringExtra(NAVIGATION_KEY) into a variable for avoiding double method call